packages/solid/src/example/muiswitch.tsx

```diff
@@ -0,0 +1,49 @@
+import styled from '@suid/material/styles/styled'
```

Comment:
Can't run a build because this component is error-ing out.
```
src/example/muiswitch.tsx(4,14): error TS2742: The inferred type of 'MaterialUISwitch' cannot be named without a reference to '.pnpm/@suid+system@0.5.2_solid-js@1.6.1/node_modules/@suid/system'. This is likely not portable. A type annotation is necessary.
```

Reply:
I'll fox these today